query_store.php

```diff
@@ -117,5 +117,15 @@
 		. urlencode( lang_get( 'query_store_error' ) );
 	print_header_redirect( $t_query_redirect_url );
 } else {
-	print_header_redirect( 'view_all_bug_page.php' );
+	# Build a redirect to view_all_set to load the filter that was saved.
+	# This will make the filter name appear as selected in the filter selection box.
+	$t_params = array(
+		'type' => 3,
+		'source_query_id' => $t_new_row_id
+	);
+	if( filter_is_temporary( $t_filter ) ) {
+		$t_params['filter'] = filter_get_temporary_key( $t_filter );
```

Comment:
Make `filter` a constant.

Reply:
We dont usually make constants for gpc parameter names. However, i see the benefit of having it as a constant.
There should be a future rewrite for how the filters are propagated through links, aiming at using filters in other contexts like summary pages, etc
I'd rather leave this point to that future work.

Reply:
ok